### DISCUSSION TITLE
New logo suggestion

### DISCUSSION THREAD

**Discussion by Donkey-Doug** [Docile-Alligator/Infinity-For-Reddit#992] [Ideas]
The current logo is not very discriptive and looks quite outdated. I made a new concept based on the R (from reddit) and the infinity symbol.

![Screenshot_20220317-232917_Draw](https://user-images.githubusercontent.com/50382414/158904985-71c2eb9f-115a-4999-8cb4-1b04339c2856.png)

Description:
The solid lines create the letter R. The dotted lines complete the infinity symbol. Optionally, two colors can be uses to accentuate the ∞ symbol (or the letter R).

**Comment by UncivilizedGhost** [comment]
The new icon should also have support for Android 12's themed icons
